packages/lib/zuauth/src/server.ts

```diff
@@ -20,9 +83,11 @@ import {
  */
 export async function authenticate(
   pcdStr: string,
-  watermark: string,
-  config: PipelineEdDSATicketZuAuthConfig[]
+  { watermark, config, fieldsToReveal, externalNullifier }: ZuAuthArgs
 ): Promise<ZKEdDSAEventTicketPCD> {
+  /**
```

Comment:
nit: I think the double `*` in `/**` here marks this as being for typedoc documentation, which doesn't seem relevant inside the body of the function.  Could be just `/*` or else `//`.

Reply:
If I don't use the double-asterisk, then VSCode does not insert a new asterisk at the beginning of the subsequent lines. From that I inferred a distinction between using `/* ... */` as a means of "commenting out", and `/** ... */` as a means of writing multi-line comments, but thiat is just my interpretation of VSCode's logic.